### PR TITLE
Search string fix

### PR DESCRIPTION
**Pull request by unka65**
This changes the search string handling to resolve issue #42.

### CODE REVIEW

File: webseer.php [hunk 1]
```diff
@@ -492,23 +493,22 @@ function clearFilter() {
...
+		<form id='form_webseer' action='webseer.php'>
+			<input type='hidden' name='search' value='search'>
 			<table class='filterTable'>
 				<tr class='noprint'>
 					<td>
 						<?php print __('Search', 'webseer');?>
 					</td>
 					<td>
-						<input type='text' id='rfilter' size='30' value='<?php print get_request_var('rfilter');?>'>
+						<input type='text' class='ui-state-default ui-corner-all' id='rfilter' size='30' value='<?php print html_escape_request_var('rfilter');?>'>
```
Comment (netniV) [inline webseer.php:511]: Aren't these added by jqueryUI ?  Not sure we should be setting them directly, was there a reason for it?

File: README.md [hunk 2]
```diff
@@ -35,7 +35,9 @@ in GitHub.
 
 ## ChangeLog
 
-* issue#41:  Maintenance schedule check add
+* issue#42: Search string fix
```
Comment (netniV) [inline README.md]: Newer issue should come below older issue

**Review by netniV — CHANGES_REQUESTED** [review]

File: webseer.php (hunk 1 above)
Comment (unka65) [reply webseer.php:511]: I do not know.  I only know to look for examples in the base code such as data_sources.php (~1426), graphs.php.  I probably copied something out of context.  I looked around but I did not find the proper method.  Where should I look for an example of what should be coded?

Comment (netniV) [reply webseer.php:511]: We can let it pass for now.

File: README.md (hunk 2 above)
Comment (unka65) [reply README.md]: It is changed.

**Review by netniV — APPROVED** [review]

**Pull request merged**